02-peer-protocol.md

```diff
@@ -829,6 +839,8 @@ A sending node:
     its commitment transaction, it cannot pay the fee for the updated local or
     remote transaction at the current `feerate_per_kw` while maintaining its
     channel reserve.
+  - SHOULD NOT offer a combination of `amount_msat`, `cltv_expiry`, `hold_fee_rate_day` and `hold_fee_discount` such that the remote node cannot pay the hold fee for the longest possible hold duration. The longest possible hold duration is the `cltv_expiry` delta in blocks multiplied by ten minutes. This must also take into account all currently outstanding htlcs.
```

Comment:
See the above comment. It seems strange to do this artificial conversion between wall clock time and block time when we already have `cltv_expiry` as the upper hold duration and a perfect time stamping server on the base layer.

Reply:
Yes, agreed that this causes friction for the calculation of the max hold fee.